plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TableStatisticsReader.java

```diff
@@ -276,41 +273,37 @@ private static Map<Integer, Long> readNdvs(Table icebergTable, long snapshotId,
     }
 
     /**
-     * Iterates over existing statistics files present for parent snapshot chain,  starting at {@code startingSnapshotId} (inclusive).
+     * Returns most recent statistics file for the given {@code snapshotId}
```

Comment:
`for the given {@code snapshotId}` -> for the specified snapshot of the table or one of its predecessors`